docs/contributing.md

```diff
@@ -227,27 +317,7 @@ Now for the process itself:
 
    Make sure to run `yarn changelog:validate` once you're done to ensure all changelogs are correctly formatted.
 
-5. **Push and submit a pull request for the release branch so that it can be reviewed and tested.**
-
-   Release PRs can be approved by codeowners of affected packages, so as long as the above guidelines have been followed, there is no need to reach out to the Wallet Framework team for approval.
-
-6. **Incorporate new changes made to `main` into changelogs.**
-
-   If at any point you see the "Update branch" button on your release PR, stop and look over the most recent commits made to `main`. If there are new changes to package you are trying to release, make sure that the changes are reflected in the changelog for that package.
-
-7. **"Squash & Merge" the release and wait for approval.**
-
-   You're almost there!
-
-   Merging triggers the [`publish-release` GitHub action](https://github.com/MetaMask/action-publish-release) workflow to tag the final release commit and publish the release on GitHub. Before packages are published to NPM, this action will automatically notify the [`npm-publishers`](https://github.com/orgs/MetaMask/teams/npm-publishers) team in Slack to review and approve the release.
-
-8. **Verify that the new versions have been published.**
-
-   Once the `npm-publishers` team has approved the release, you can click on the link in the Slack message to monitor the remainder of the process.
-
-   Once the action has completed, [check NPM](https://npms.io/search?q=scope%3Ametamask) to verify that all relevant packages has been published.
-
-   You're done!
+5. **Follow steps 6-9 from Option A above to complete the release.**
```

Comment:
I guess this is okay, but it seems odd to jump to another workflow to complete this workflow. Maybe it's easiest to repeat the steps here for now, and we can refactor this later?